run.sh

```diff
@@ -74,7 +74,8 @@ ID_FS_LABEL=${ID_FS_LABEL:-$(blkid -o value -s LABEL)}
 ID_FS_UUID=${ID_FS_UUID:-$(blkid -o value -s UUID)}
 TEMPLATE="${ID_FS_LABEL:-nolabel}_${ID_FS_UUID:-nouuid}_XXX"
 
-DIRECTORY=$(mktemp -d "/output/$TEMPLATE")
+mkdir -p /output/.makemkv/
+DIRECTORY=$(mktemp -d "/output/.makemkv/$TEMPLATE")
```

Comment:
The problem with this there's an edge case where mktemp creates a directory in /output/.makemkv that collides with one in /output. Maybe a better approach is to just add the `sudo -u mkv mkdir "$DIRECTORY/.rip"` and add /.rip to the makemkvcon line, then move all files up one dir and rmdir .rip when done.